### PR TITLE
[Chore] Golang update to 1.19

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.19
       - run: GOPROXY=https://proxy.golang.org,direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./kuma/... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
```
Comment (acald-creator) [inline .github/workflows/ci.yml:27]: We will need to update this to `golangci-lint`

See https://github.com/meshery/meshery-istio/pull/490 for comparison.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.19
       - run: GOPROXY=https://proxy.golang.org,direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -tags draft ./...
```
Comment (acald-creator) [inline .github/workflows/ci.yml:40]: `go get -u` has been deprecated

See previous PR from `meshery-istio` for comparison

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.19
       - run: GOPROXY=https://proxy.golang.org,direct GOSUMDB=off GO111MODULE=on go get -u honnef.co/go/tools/cmd/staticcheck; /home/runner/go/bin/staticcheck -tags draft -checks all ./kuma/... # https://staticcheck.io/docs/checks
```
Comment (acald-creator) [inline .github/workflows/ci.yml:75]: Same thing here.

**Review by acald-creator — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (leecalcote) [reply .github/workflows/ci.yml:27]: Makes much sense. Please do carry this PR forward, if you're able to make commits on the PR/branch.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
```
Comment (leecalcote) [inline .github/workflows/ci.yml]:
```suggestion
        uses: actions/setup-go@v3
```

File: .github/workflows/ci.yml (hunk 3 above)
Comment (leecalcote) [reply .github/workflows/ci.yml:84]: thank you for this.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (leecalcote) [reply .github/workflows/ci.yml:49]: I believe that I have addressed this now.

**Comment by acald-creator** [comment]
@leecalcote working on it

**Comment by acald-creator** [comment]
Should be good now.

**Review by Revolyssup — APPROVED** [review]

**Comment by Revolyssup** [comment]
@leecalcote  @acald-creator  LGTM! Anything else from your side?

**Comment by acald-creator** [comment]
lgtm, nearly forgot this was one of the few that was still running on go 1.17

**Comment by leecalcote** [comment]
Thank you!

**Pull request merged**